manifests/params.pp

```diff
@@ -19,6 +19,9 @@
   $install_java               = false
   $java_version               = '8'
   $java_home                  = '/usr/jdk64/jdk1.8.0_77'
+  $spark_home                 = '/usr/hdp/2.5.3.0-37/spark'
+  $hive_home                  = '/usr/hdp/2.5.3.0-37/hive'
+  $hcat_home                  = '/usr/hdp/2.5.3.0-37/hive-hcatalog'
```

Comment:
What are those version numbers hardcoded there, do they come from another variable?

Reply:
After the explanation of Axier, it looks good for me...seeing that puppet will dead after K8s in the following month.
LGTM